run_node.sh

```diff
@@ -65,16 +66,24 @@ while [[ $# -gt 0 ]]; do
         --sync_from_genesis)
             SYNC=true
             shift;;
+        --stash_account)
+            STASH_ACCOUNT=$2
+            shift 2;;
         -* | --* )
             echo "Warning: unrecognized option: $1"
-            exit;; 
+            exit;;
         *)
             echo "Unrecognized command"
             Help
             exit;;
   esac
 done
 
+if [[ -z "$ARCHIVIST" ]]
+then
+    STASH_ACCOUNT=${STASH_ACCOUNT:?'Please specify your stash account_id.'}
```

Comment:
Maybe we should make this argument mandatory and reflect this in the `usage` docstring?